torchvision/datasets/utils.py

```diff
@@ -59,15 +108,26 @@ def calculate_md5(fpath: str, chunk_size: int = 1024 * 1024) -> str:
 
 
 def check_md5(fpath: str, md5: str, **kwargs: Any) -> bool:
+    """DEPRECATED"""
```

Comment:
we might need to raise a proper warning to indicate the deprecation. something like https://github.com/pytorch/vision/blob/master/torchvision/transforms/functional_tensor.py#L267:L267

(same above)